nx.json

```diff
@@ -30,6 +30,35 @@
     "e2e": {
       "dependsOn": ["^build"],
       "inputs": ["default", "^production"]
+    },
+    "version": {
+      "dependsOn": ["^build"],
+      "inputs": ["production", "^production"],
+      "executor": "@jscutlery/semver:version",
+      "options": {
+        "commitMessageFormat": "release(${projectName}): ${version}",
+        "tagPrefix": "{projectName}@",
+        "preset": {
+          "name": "conventionalcommits",
+          "commitUrlFormat": "{{host}}/{{owner}}/{{repository}}/commit/{{hash}}",
+          "compareUrlFormat": "{{host}}/{{owner}}/{{repository}}/compare/{{previousTag}}...{{currentTag}}",
+          "issueUrlFormat": "{{host}}/{{owner}}/{{repository}}/issues/{{id}}",
+          "types": [
+            { "type": "feat", "section": "Features" },
+            { "type": "fix", "section": "Bug Fixes" },
+            { "type": "build", "section": "Dependency updates" },
+            { "type": "docs", "section": "Documentation" },
+            { "type": "refactor", "section": "Code Refactoring" },
+            { "type": "perf", "section": "Performance Improvements" }
+          ]
+        }
+      }
```

Comment:
Maybe a stupid question, but... do we need to specify all this `preset` on such detail? We already define `commitMessageFormat` explicitly, so what is it used for?

In the [`portal-client`'s `version` executor](https://github.com/code-pushup/portal/blob/main/libs/portal-client/project.json#L64-L68), I only needed to specify `"preset": "conventional"`. That's much less configuration to maintain. :man_shrugging:  (The commit messages are then `chore(portal-client): release version 0.1.2` and the tag is [`portal-client-0.1.2`](https://github.com/code-pushup/portal/releases/tag/portal-client-0.1.2), but I assume only `commitMessageFormat` and `tagPrefix` are needed to customize that).

Reply:
I will check again and come back with a explanation

Reply:
I was able to get rid of some options, but not all...


**By default it is:**

```md
---
## [0.0.2](https://github.com/flowup/quality-metrics-cli/compare/models@0.0.1...models@0.0.2) (2023-11-16)
---
```

**With config it is:**
```md
---
## [0.0.2](https://github.com/flowup/quality-metrics-cli/compare/models@0.0.1...models@0.0.2) (2023-11-16)


### Code Refactoring

* revert ([78d8154](https://github.com/flowup/quality-metrics-cli/commit/78d8154d8494d205b729b2f83ee29c6299b3280e))
---

```